ta/pkcs11/src/attributes.c

```diff
@@ -254,7 +254,9 @@ static void __trace_attributes(char *prefix, void *src, void *end)
 		case PKCS11_CKA_WRAP_TEMPLATE:
 		case PKCS11_CKA_UNWRAP_TEMPLATE:
 		case PKCS11_CKA_DERIVE_TEMPLATE:
-			trace_attributes(prefix2, cur + sizeof(pkcs11_ref));
+			if (pkcs11_ref.size)
```

Comment:
Wondering should we trace also these cases to trace log but just have "empty". Otherwise it might make debugging a problem a bit harder?

Reply:
Ok, I'll change to add a specific trace when such attribute is empty.

Reply:
Actually the change is fine as is. The (empty) attribute is already printed out (with a null size info) in the switch/case right above.